Feature

gituser768 opened the pull request:
I've simply added:
-Transpose protocol implementation for complex matrices
-Scaling by a complex valued number
-A small change to improve error handling

gituser768 committed:
Added Transpose protocol

gituser768 committed:
Added error handling

gituser768 committed:
Added ability to scale using by a complex number

gituser768 committed:
Fixed scaling by complex number

mikera commented:
Hi @dmh43, looks sensible.... can you add in some tests for the functionality you have added / changed?

gituser768 commented:
Yes, I'll do that and and commit those additions.

gituser768 committed:
Added tests for:
-scaling by a complex value
-taking the transpose of a complex valued matrix

Will add generative testing in the future
